src/shared/sol-random.c

```diff
@@ -86,7 +86,9 @@ getrandom_shim(void *buf, size_t buflen, unsigned int flags)
 static uint64_t
 get_platform_seed(uint64_t seed)
```

Comment:
Doesn't RIOT have anything to generate some entropy? (Maybe we could test if `RDRAND` is available?)